diffusion/models/models.py

```diff
@@ -187,7 +205,8 @@ def stable_diffusion_xl(
     autoencoder_path: Optional[str] = None,
     autoencoder_local_path: str = '/tmp/autoencoder_weights.pt',
     prediction_type: str = 'epsilon',
-    latent_scale: Union[float, str] = 0.13025,
+    latent_mean: Union[float, Tuple, str] = 0.0,
+    latent_std: Union[float, Tuple, str] = 0.13025,
```

Comment:
Should this be 1/0.13025?

Reply:
Yes, it should. Nice catch! Fixed it now.